test/Reader/Http/Psr7ResponseDecoratorTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendTest\Feed\Reader\Http;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Psr\Http\Message\ResponseInterface as Psr7ResponseInterface;
+use Zend\Feed\Reader\Http\Psr7ResponseDecorator;
+use Zend\Feed\Reader\Http\HeaderAwareResponseInterface;
+use Zend\Feed\Reader\Http\ResponseInterface;
+use ZendTest\Feed\Reader\TestAsset\Psr7Stream;
+
+class Psr7ResponseDecoratorTest extends TestCase
```

Comment:
> HeaderAwareResponseInterface tests could be extracted to a common trait 

I've opted not to do this in my current update, as there are significant differences, and little overlap in testing strategy between the tests of the two concrete response types.